docs/screencast-pmem.sh

```diff
@@ -0,0 +1,415 @@
+  out ''
+  sleep 1
+  command 'cat subscription-pmem-csi.yaml'
+  sleep 2
+  command 'kubectl apply -f subscription-pmem-csi.yaml'
+  sleep 5
+  command 'kubectl wait --for condition=available deployment/pmem-csi-operator -n pmem-csi'
+  sleep 1
+  out '#  Verify the operator was installed and is running.'
+  out ''
+  sleep 1
+  command 'kubectl get pods -n pmem-csi'
+  sleep 2
+  out '#  Success!'
+  sleep 3
+}
+
+screen4() # Create pmemcsideployment
+{
+  clear
+  prompt
+  out '#  Now a driver deployment needs to be created using a pmemcsideployment crd.'
+  out ''
+  sleep 1
+  command 'cat pmemcsideployment.yaml'
+  sleep 2
+  command 'kubectl apply -f pmemcsideployment.yaml'
+  sleep 2
+  command 'kubectl get pmemcsideployment'
+  sleep 2
+  out '#  Check to see if the controller and node pods were created.'
+  out ''
+  sleep 1
+  command 'kubectl wait --for condition=available deployment/pmem-csi-intel-com-controller -n pmem-csi'
+  sleep 2
+  command 'kubectl get pods -n pmem-csi'
+  sleep 2
+  out '#  GREAT! Now the driver is running and ready to create persistent volumes.'
+  sleep 3
+}
+
+screen5()  # Create a StorageClass and PVC
+{
+  clear
+  prompt
+  out '#  Next, a storage class needs to be created. This one uses a volumeBindingMode'
+  out '#  called WaitForFirstConsumer which means that volumes will not be created'
+  out '#  until a pod really needs them. This is better for pod scheduling.'
+  out ''
+  sleep 1
+  command 'curl -L https://github.com/intel/pmem-csi/raw/v1.0.1/deploy/common/pmem-storageclass-late-binding.yaml'
+  sleep 2
+  command 'kubectl apply -f https://github.com/intel/pmem-csi/raw/v1.0.1/deploy/common/pmem-storageclass-late-binding.yaml'
+  sleep 2
+  out '#  Create a Persistent Volume Claim for that storage class.'
+  out ''
+  sleep 1
+  command 'curl -L https://github.com/intel/pmem-csi/raw/v1.0.1/deploy/common/pmem-pvc-late-binding.yaml'
+  sleep 2
+  command 'kubectl apply -f https://github.com/intel/pmem-csi/raw/v1.0.1/deploy/common/pmem-pvc-late-binding.yaml'
+  sleep 2
+  out '#  If you describe the PVC you will see the volume is still waiting for its first consumer.'
+  out ''
+  sleep 1
+  command 'kubectl describe pvc/pmem-csi-pvc-late-binding'
+  sleep 3
+}
+
+screen6()  # Start pod app and show users that PMem is being used
+{
+  clear
+  prompt
+  out '#  To check that everything is working properly, deploy a pod that will mount persistent memory'
+  out '#  to the /data partition of the container.'
+  out ''
+  sleep 1
+  command 'curl -L https://github.com/intel/pmem-csi/raw/v1.0.1/deploy/common/pmem-app-late-binding.yaml'
+  sleep 2
+  command 'kubectl apply -f https://github.com/intel/pmem-csi/raw/v1.0.1/deploy/common/pmem-app-late-binding.yaml'
+  sleep 2
+  command 'kubectl wait --for condition=ready pod/my-csi-app'
+  sleep 1
+  command 'kubectl get pods'
+  sleep 2
+  out '#  Inside this pod there is an application called pmem-dax-check which can verify if'
```

Comment:
Not sure about this one...

```suggestion
  out '#  Inside this pod there is an application called pmem-dax-check which can verify whether'
```